test/gleam/string_test.gleam

```diff
@@ -779,3 +780,10 @@ if erlang {
     |> should.equal(True)
   }
 }
+
+pub fn inspect_bit_string_test() {
+  "Hello from Gleam!"
+  |> bit_string.from_string()
+  |> string.inspect()
+  |> should.equal("Hello from Gleam!")
```

Comment:
This test should use conditional compilation as the expected result is not the same on Erlang as on JS.

Reply:
Why is the expected result not the same?

Reply:
Because the runtime representation for strings and bit strings on Erlang are the same while they are different on JavaScript. Given an Erlang binary with no type information we cannot know if it is a string or a bit string.

Reply:
Thanks!